sensorhub-service-consys/src/main/java/org/sensorhub/impl/service/consys/obs/ObsBindingOmJson.java

```diff
@@ -109,7 +109,7 @@ public IObsData deserialize(JsonReader reader) throws IOException
                     obs.withPhenomenonTime(OffsetDateTime.parse(reader.nextString()).toInstant());
                 else if ("resultTime".equals(propName))
                     obs.withResultTime(OffsetDateTime.parse(reader.nextString()).toInstant());
-                else if ("foi@id".equals(propName))
+                else if ("foi@id".equals(propName) && !ctx.isClientSide())
```

Comment:
We need at least this ID on the client side too.
The client needs to be able to parse it and also assign an FOI to an observation when posting it to the server.